plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HivePrivilegeInfo.java

```diff
@@ -84,6 +84,9 @@ public HivePrincipal getGrantee()
     public static HivePrivilege toHivePrivilege(Privilege privilege)
     {
         switch (privilege) {
+            case CREATE:
```

Comment:
The break makes this a bit hard to follow, since this case is actually redundant. Maybe put it at the end
```java
case CREATE: // not supported by Hive
default:
    throw new IllegalArgumentException("Unexpected privilege: " + privilege);

Reply:
Grouping it with `default`, on the other hand, prevents Error Prone from checking for exhaustiveness of the switch